cnf-tests/Dockerfile.openshift

```diff
@@ -1,5 +1,5 @@
 # This dockerfile is specific to building the OpenShift CNF tests image
-FROM openshift/origin-release:golang-1.16 as builder
+FROM openshift/origin-release:golang-1.17 as builder
```

Comment:
probably we should use `registry.ci.openshift.org/openshift/release:golang-1.17 AS builder` at least it what the OpenShift is using

Reply:
uhm, should we do the jump in this PR or with yet another one? (cc @SchSeba @fedepaol )

Reply:
I think we can leave it like that and maybe bump it into a future PR.

Reply:
@SchSeba I think openshift does not support anymore openshift/origin-release because the latest update for images was one year ago.
I think we should change the base image to `registry.ci.openshift.org/openshift/release:golang-1.17 AS builder`